src/main/java/org/fogbowcloud/app/jes/arrebol/models/ArrebolTaskState.java

```diff
@@ -0,0 +1,5 @@
+package org.fogbowcloud.app.jes.arrebol.models;
+
+public enum ArrebolTaskState {
+    PENDING, FAILED, FINISHED, RUNNING, CLOSED
```

Comment:
@paulofelipefeitosa and @wesleymonte, do you need this CLOSED state in iguassu?

Reply:
Nope. This state has been removed.